build_scripts/build_runner.py

```diff
@@ -771,9 +785,10 @@ def main():
     try:
         build_config.generate_build_config()
         build_config.run_stage(args.stage)
-    except Exception:
+    except Exception as exc:
         set_output_stream('err')
-        log.exception("Exception occurred")
+        if not isinstance(exc, subprocess.CalledProcessError):
+            log.exception("Exception occurred")
         log.error("%sING FAILED", args.stage.name)
         with open(pathlib.Path(args.root_dir) / 'build_state', 'w') as status:
             status.write(json.dumps({'status': "FAIL"}))
```

Comment:
pathlib.Path objects have **write_text** method. I would recommend using it.

Reply:
@dimakozlov Can you explain me why write_text method is preferable to use? :)

Reply:
Less code and more understandable )